packages/sdk/src/impl/data/infrastructure/jwt/JwtServiceImpl.ts

```diff
@@ -48,24 +39,36 @@ export default class JwtServiceImpl implements JwtService {
             typ: "JWT",
         };
 
-        new SignJWT(jwtDescriptor.payload)
+        let signedJwtRes = new SignJWT(jwtDescriptor.payload)
             .setProtectedHeader(header)
             .setAudience(jwtDescriptor.iss)
             .setIssuer(jwtDescriptor.iss)
             .setJti(jwtDescriptor.jti)
-            .setIssuedAt(Date.now())
-            .setNotBefore(Date.now())
-            .setExpirationTime(new Date().addDaysToNow(7).getTime())
+            .setIssuedAt(undefined)
```

Comment:
`setIssuedAt(undefined)`
`setNotBefore(0)`
both values should be current time - see Android implementation.
@anilasimbilen

Reply:
Yes, the default value for issuedAt is current time at jose library.